book/src/data-model/sboms.md

```diff
@@ -0,0 +1,18 @@
+## SBOMs
+
+The `Sbom` entity records that an SBOM was ingested or generated for a `Package` that is being
+_directly monitored_. See the [`Packages`](./packages.md) documentation for a discussion.
+
+![SBOMs](sboms.png)
+
+### Data Dictionary
+
+- `Sbom` - An SBOM is a snapshot inventory of the components that make up a piece of software at a 
+    moment in time.
+- `Package` - An inventory item from an `Sbom`. Can be defined by standard identifiers like CPE 
+  or PURL.
+- `Vulnerability` - An identified security vulnerability related to a `Package`.
+- `Author` - The actor that provided or generated the Sbom. For vendor supplied SBOMS, this 
```

Comment:
I forget when we brought `Author` in as a term.  Where does it come from?  I'm not sure it's different from the provider.

Reply:
It's from the "NTIA Minimum Elements of an SBOM". You can refer to that doc for the exact definition.